src/components/PublicHeader.tsx

```diff
@@ -10,15 +10,17 @@ import {
 } from '@adobe/react-spectrum';
 
 import apiUrl from '../utils/apiUrl';
-import { ZetkinOrganization, ZetkinUser } from '../types/zetkin';
+import { ZetkinMembership, ZetkinOrganization, ZetkinUser } from '../types/zetkin';
 
 interface PublicHeaderProps {
     user: ZetkinUser | null;
     org?: ZetkinOrganization | null;
     canOrganize?: boolean;
+    officials: ZetkinMembership[];
 }
 
-const PublicHeader = ({ canOrganize, user, org  } : PublicHeaderProps) : JSX.Element => {
+const PublicHeader = ({ canOrganize, user, org, officials: memberships  } : PublicHeaderProps) : JSX.Element => {
```

Comment:
The reassignment of `officials` to `memberships` here only strengthens my opinion from the previous comment that the property should be renamed to `officialMemberships` (if filtered outside) or `userMemberships` (if filtered within this component).

Avoid reassignment like this when possible. When reading the component after reading the interface, I was stumped to come across the `memberships variable` further down, until I realized it was reassigned from `officials` here. Having read the interface I knew there was no property called `memberships` and I didn't see one being declared, because it's not obvious from just glancing over a function signature containing object destructuring.